CONTRIBUTING.md

```diff
@@ -0,0 +1,107 @@
+<h1>Contributing to JobiE</h1>
+<p>:wave: Hope you enjoy contributing to jobie :+1:</p>
+The following is a set of guidelines to help you begin your contribution to the JobiE. If you want to propose any change to this document feel free to propose changes to this document in a pull request.
+
+<h1>How to Contribute</h1>
+
+<h2>Raise an Issue</h2>
+<p>Did you find something which can be improved further ?? Feel free to share it <a href="https://github.com/akshatnema/jobie/issues"> here</a>.</p>
+
+<p>Pull requests are the way concrete changes are made to the code, documentation dependencies, and tools contained in the JobiE repository.</p>
+<h2>Setting up your local environment</h2>
+<h3>Step 1:Fork</h3>
+<p>Fork the project on Github and clone your fork locally to your system.</p>
+
+```console
```

Comment:
Don't merge the commands for ssh and HTTPS. Add them separately as:
Using SSH
```bash
  git clone git@github.com:<your_username>/jobie.git
```
OR
Using HTTPS:
```bash
git clone https://github.com/<your_username>/jobie.git
```